### DISCUSSION TITLE
Manual/class-Light

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#9] [Notes]
# Manual/class-Light

https://docs.unity3d.com/Manual/class-Light.html

**Comment by vertxxyz** [comment]
When trying to isolate a small amount of layers in a **Culling Mask** via the inspector it's important to note that toggling the mask to *Nothing* will set **all** the layers to none, including the hidden un-bound layers.
This will cause warnings as new layers are created if running in [deferred](https://docs.unity3d.com/Manual/RenderTech-DeferredShading.html), where you can only exclude a limited amount of layers from from culling masks (4).
To exclude layers you should start from *Everything* and work backwards.
